test/parallel/test-process-startTime.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+const now = Date.now();
+const diffFromNow = now - process.startTime;
+
+// Make it a bit flexible
+assert.ok(diffFromNow > 0);
+assert.ok(diffFromNow < common.platformTimeout(100));
```

Comment:
Would it be acceptable to remove this line entirely and rely on the previous assertion only? Arbitrary timing--"this shouldn't take longer than 500 milliseconds!"--has been the source of far too many test problems. /cc @nodejs/testing